src/App.jsx

```diff
@@ -14,57 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
```

Comment:
Hello Mariya!!! Looks great. But we discus about that it's not good idea for take goods index. If you look in checklist we must save a selectedGood in the state (Jam is the default value).

Reply:
Hi! Goods[8] is just a default value (it is 'Jam')
Its the same as if I just wrote 'Jam' literally